controllers/controllers.js

```diff
@@ -54,3 +55,20 @@ exports.getCommentsByArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const newComment = req.body;
+
+  const articlePromises = [insertComment(article_id, newComment)];
+
+  if (article_id) {
```

Comment:
Why would the article id not exist on this end point? This lends more to optional queries as per Stephens example with topics. Article id should always exist on this end point.